frontend/dispatcher.lua

```diff
@@ -547,7 +547,11 @@ function Dispatcher:execute(ui, settings, gesture)
     for k, v in pairs(settings) do
         if settingsList[k].conditions == nil or settingsList[k].conditions == true then
             if settingsList[k].category == "none" then
-                ui:handleEvent(Event:new(settingsList[k].event))
+                if settingsList[k].arg ~= nil then
+                    ui:handleEvent(Event:new(settingsList[k].event, settingsList[k].arg))
```

Comment:
It's probably not relevant here or we would've noticed already, but of course there are also events with multiple arguments. For that you could use probably use `table.unpack()` when necessary.

Reply:
not relavant here, and perfer to keep the code simpler for now.